widget/led.go

```diff
@@ -0,0 +1,145 @@
+package widget
+
+import (
+	"image/color"
+	"log"
+
+	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/canvas"
+	"fyne.io/fyne/v2/container"
+	"fyne.io/fyne/v2/widget"
+)
+
+var defaultLedOnColor color.Color = color.RGBA{255, 0, 0, 255}
+var defaultLedOffColor color.Color = color.RGBA{120, 120, 120, 255}
+
+type ledRenderer struct {
+	led    *Led
+	circle *canvas.Circle
+	text   *canvas.Text
+}
+
+func (h *ledRenderer) MinSize() fyne.Size {
+	log.Println(h.led.Size())
+	return fyne.NewSize(h.led.Size().Width, h.led.Size().Height)
```

Comment:
I'm not sure if the current size should be the minimum but this is at least simpler than what is there currently:
```suggestion
	return h.led.Size()
```